libraries/AP_HAL_ChibiOS/hwdef/omnibusf4/hwdef.dat

```diff
@@ -0,0 +1,14 @@
+# hw definition file for processing by chibios_pins.py
+# Omnibus F4 AIO (no current sensor onboard) boards
+# with F405 mcu, mpu6000 imu, bmp280 barometer, 7456 series osd  and sdcard
+
+include ../omnibusf4pro.dat
+
+undef PC8
+undef PC9
+
+#Omnibus F4 AIO V3 and later had hw inverter on UART6
+#Overide it to use as GPS UART port
+PC8 SBUS_INVERT_RX OUTPUT LOW
+PC9 SBUS_INVERT_TX OUTPUT LOW
+
```

Comment:
BOARD_PWM_COUNT_DEFAULT probably also should be redefined

Reply:
will do...also I found a conflict with RSSI which I corrected...